Dockerfile

```diff
@@ -1,21 +1,27 @@
-FROM buildkite/puppeteer:latest
+# Use node 4.4.5 LTS
```

Comment:
Is this accurate? This would be a very old version of node…

Reply:
Oh so it's not node 4.5, but 14.

Reply:
That node line is commented out. I think I tried it for a minute, but ended up going back to using buildkite/puppeteer, which is based on node14